test/e2e/frontend/cypress/tests/applications/overview.spec.js

```diff
@@ -1148,5 +1148,106 @@ describe('FlowForge - Applications', () => {
                     })
             })
         })
+
+        describe('device kebab menu', () => {
+            const MENU_ITEMS = [
+                {
+                    index: 0,
+                    label: 'Edit Details',
+                    dialogTitle: 'Update Device',
+                    dialogDataEl: 'team-device-create-dialog'
+                },
+                // This item will be enabled once #4249 is resolved
+                // {
+                //     index: 1,
+                //     label: 'Remove From Application',
+                //     dialogTitle: 'Remove Device from Application',
+                //     dialogDataEl: 'platform-dialog'
+                // },
```

Comment:
is there a commit/merge order I need to be aware of with PRs?

Reply:
no, this works as is Joe.

Once c is resolved, these comments will be un-done and the test for missing item will be caught,

As things stand, the "Remove from application" is missing due to how/where the device data is gotten from the API. It is essentially missing the `owner` prop so the menu item is not shown.  The tests here, as they stand reflect the current state of affairs. 

The commented code is how it SHOULD be (and will be once #4249 is resolved)

Reply:
To be clear, assuming this passes it should be reviewed/merged immediately to fix the live issue.  The missing entry is less important and can be fixed at a later time.

I will make reference to these test in the issue.